janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -961,4 +964,11 @@ private AdminMask getAdminInterface() {
             throw new JanusGraphException(e);
         }
     }
+
+    private String setTableName(org.janusgraph.diskstorage.configuration.Configuration config) {
```

Comment:
This doesn't set anything. Should be renamed according to what it does. `determineTableName`?